src/resources/styles.ts

```diff
@@ -59,6 +60,10 @@ export const haStyle = css`
     background-color: var(--primary-background-color);
   }
 
+  app-toolbar {
+    box-sizing: border-box;
```

Comment:
Do we need this without the border?

Reply:
Nope... but thought lets only remove border-bottom from the app-toolbar... 
Removed complete app-toolbar code block is added.